framework/core/src/Install/Steps/EnableBundledExtensions.php

```diff
@@ -24,22 +24,7 @@
 
 class EnableBundledExtensions implements Step
 {
-    public const EXTENSION_WHITELIST = [
-        'flarum-approval',
-        'flarum-bbcode',
-        'flarum-emoji',
-        'flarum-lang-english',
-        'flarum-flags',
-        'flarum-likes',
-        'flarum-lock',
-        'flarum-markdown',
-        'flarum-mentions',
-        'flarum-statistics',
-        'flarum-sticky',
-        'flarum-subscriptions',
-        'flarum-suspend',
-        'flarum-tags',
-    ];
+    public const EXTENSION_WHITELIST = [];
```

Comment:
Unrelated, but maybe we should rename this to `EXTENSION_ALLOWLIST` or `DEFAULT_ENABLED_EXTENSIONS`